nextcloudappstore/user/signals.py

```diff
@@ -26,9 +27,20 @@ def password_changed_signal(sender, instance, **kwargs):
     :param kwargs:
     :return:
     """
+    from_mail = 'appstore@nextcloud.com'
+    mail_subect = 'Nextcloud Appstore password changed'
```

Comment:
Both messages should be translated

Reply:
since i'm not familiar with the translation, could you please provide a place so i can read about it?

Reply:
import here: https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/models.py#L16

usage here: https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/models.py#L60